hw/ip/hmac/data/hmac_testplan.hjson

```diff
@@ -59,6 +59,15 @@
       milestone: V2
       tests: ["hmac_datapath_stress"]
     }
+    {
+      name: error
+      desc: '''This error case runs sequences that will cause interrupt bit hmac_err to set.
+            This sequence includes:
+            - Write msg_fifo when sha is disable
```

Comment:
```suggestion
            - Write msg_fifo when sha is disabled
```

Reply:
Thanks Philipp for the good catch, updated!